src/Wallet/KeyFile.purs

```diff
@@ -45,3 +53,38 @@ privateStakeKeyFromFile filePath = do
   bytes <- keyFromFile filePath StakeSigningKeyShelleyed25519
   liftM (error "Unable to decode private stake key") $
     PrivateStakeKey <$> privateKeyFromBytes (wrap bytes)
+
+privatePaymentKeyToFile :: FilePath -> PrivatePaymentKey -> Aff Unit
+privatePaymentKeyToFile filePath key =
+  liftM (error "failed to format key") (formatPaymentKey key) >>=
+    liftEffect <<< (writeTextFile Encoding.UTF8 filePath)
+
+privateStakeKeyToFile :: FilePath -> PrivateStakeKey -> Aff Unit
+privateStakeKeyToFile filePath key =
+  liftM (error "failed to format key") (formatStakeKey key) >>=
+    liftEffect <<< (writeTextFile Encoding.UTF8 filePath)
+
+formatPaymentKey :: PrivatePaymentKey -> Maybe String
+formatPaymentKey (PrivatePaymentKey key) = encodeAeson >>> show
+  <$>
+    { "type": "PaymentSigningKeyShelley_ed25519"
+    , description: "Payment Signing Key"
+    , cborHex: _
+    }
+  <$> keyToCbor key
+
+formatStakeKey :: PrivateStakeKey -> Maybe String
+formatStakeKey (PrivateStakeKey key) = encodeAeson >>> show
+  <$>
+    { "type": "StakeSigningKeyShelley_ed25519"
+    , description: "Stake Signing Key"
+    , cborHex: _
+    }
+  <$> keyToCbor key
+
+keyToCbor :: PrivateKey -> Maybe String
+keyToCbor = ((rawBytesToHex >>> (magicPrefix <> _)) <$> _) <<<
+  bytesFromPrivateKey
+
+magicPrefix :: String
+magicPrefix = "5820"
```

Comment:
@klntsky Do you know what this comes from? It's needed to pass the tests but I have no idea why.

Reply:
It's from CBOR. 58 is a tag for array, 20 is byte length.

Reply:
Oh cool, thanks.